tools/view_model.cpp

```diff
@@ -206,7 +220,7 @@ void CallBackFunc(int event, int x, int y, int flags, void* userdata)
         }
         else if (flags & cv::EVENT_FLAG_MBUTTON)
         {
-            cam_dist += cam_dist * dy * 0.002;
+            cam_dist += cam_dist * dy * 0.003;
```

Comment:
Any chance of explaining what the magic constant is about? Or better, make a constant with a proper name for it?

Reply:
I don't know :)